tools/apiview/parsers/apiview-treestyle-parser-schema/model.json

```diff
@@ -143,14 +151,16 @@
             "type": "object",
             "properties": {
                 "DiagnosticId": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Diagnostic ID is auto generated ID by CSharp analyzer."
```

Comment:
oh interesting. this means that the Json file wasn't updated last time you changed the TSP file.
Could we add a CI check that will validate that the current json matches the TSP file?

Reply:
I will add CI in a separate PR to generate Json from TypeSpec and compare.